### DISCUSSION TITLE
How to change/add specific actions

### DISCUSSION THREAD

**Discussion by mischabraam** [community/community#56122] [Actions] [answered]
### Select Topic Area

Question

### Body

In my public free repository, which is around for several years, I have a couple of 3rd party actions in my 'allow list'. They are configured in Actions > General > "Allow <org>, and select non-<org>, actions and reusable workflows" > "Allow specific actions and reusable workflows". See screenshot.

![Screenshot 2023-05-22 at 15 47 24](https://github.com/community/community/assets/19824986/9e5143c3-ef49-4791-9f2c-f59f31a0032d)

However, I can't add a new 3rd party action here? I'm an owner of the repository and part of this organization. Do I need to configure something else first?

**Comment by airtower-luna** [comment, marked as answer]
I notice the repository belongs to an organization, not your user account. Which actions can be used might be limited at the organization level, in that case it'd need to be allowed there: https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization#allowing-select-actions-and-reusable-workflows-to-run

**Comment by mischabraam** [comment]
Awh man, thanks!
